requirements/development.txt

```diff
@@ -192,7 +192,7 @@ protobuf==4.23.0
     #   googleapis-common-protos
     #   grpcio-status
     #   proto-plus
-psycopg2-binary==2.9.6
+psycopg2-binary==2.9.9
```

Comment:
this file is dynamically generated from running something like `pip-compile-multi -P psycopg2-binary`, but first you'll have to unpin it here -> https://github.com/apache/superset/blob/master/pyproject.toml#L153

I'm not sure why it's pinned currently, but I'd suggest changing that line to:
`postgres = ["psycopg2-binary>=2.9.6"]` and then running the command above.